worldedit-core/src/main/java/com/sk89q/worldedit/regions/selector/ExtendingCuboidRegionSelector.java

```diff
@@ -122,9 +122,9 @@ public boolean selectSecondary(BlockVector3 position, SelectorLimits limits) {
         region.setPos1(position1);
         region.setPos2(position2);
 
-        assert(region.contains(o1));
-        assert(region.contains(o2));
-        assert(region.contains(position));
+        assert (region.contains(o1));
```

Comment:
Is this a method call? Or do we actually have assert statements in our code here

Reply:
These are assert statements. I suppose we could remove the `()` for clarity.

Reply:
I have done so.